### PR TITLE
fix: filter by rating , added  react toastify 

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
I have added react toastify and fixed filter clear button , that you can ignore . 
Also added filter by rating , so please check the below mentioned files , and ignore other files 
files to check 
src/context/WishlistContext.js
src/components/sidebar/Sidebar.jsx
src/filterFunction/categoryFunction.js

image

### CODE REVIEW

**Review by arjun-computer-geek — APPROVED** [review]
Looks good good code of conduct. Go and merge🔥🔥

File: src/components/product-card/ProductCard.jsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React  from 'react'
 import { useWishList} from 'context/WishListContext' 
 import "./ProductCard.css"
+// import { toast } from 'react-toastify';
```
Comment (Jia-A) [inline src/components/product-card/ProductCard.jsx]: Remove comments and extra spacing.

**Review by Jia-A — APPROVED** [review]
Good naming conventions
You can format your code with Beautify or Prettier.
Everything else looks nice, good to merge.

File: src/components/product-card/ProductCard.jsx (hunk 1 above)
Comment (Shubhamkr2610) [reply src/components/product-card/ProductCard.jsx]: removed

**Pull request merged**